tests/regression/cpanel-plugin/9510100.yaml

```
@@ -1,23 +1,23 @@
 ---
 meta:
   author: "Felipe Zipitria, azurit"
-  description: "CPanel Plugin"
+  description: "cPanel Rule Exclusions Plugin"
   enabled: true
   name: 9510100.yaml
```

Oops. Merged too soon @EsadCetiner.

```suggestion
rule_id: 9510100
```